Assets/Engine/Interactions/InteractionHandler.cs

```diff
@@ -98,14 +100,13 @@ public void Update()
         [Command]
         private void CmdRunInteraction(Ray ray, int index, string name)
         {
-            var chosenInteraction = GetViableInteractions(ray)[index];
-
-            // Ensure the interaction can happen
-            if (!chosenInteraction.CanInteract())
```

Comment:
CanInteract should still be called, to prevent the client sending an interaction that the client shouldn't actually be able to perform.

Reply:
GetViableInteractions already filters by CanInteract

Reply:
Oh lol, that must've been my bad then